MetaMask/metamask-mobile
# [FIX] Fix 6038 null address issue

**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

This fixes the issue related to a stripHexPrefix error - https://github.com/MetaMask/metamask-mobile/issues/6038

How to test:
- Go to app.manifold.xyz
- Initiate a mint
- Confirm the signature
- Metamask should not throw an issue

**Checklist**
